pylintrc

```diff
@@ -3,7 +3,7 @@ ignore-patterns=test_*,conftest,setup
 reports=no
 
 # PYLINT DIRECTORY BLACKLIST.
-ignore=_vendor,_generated,samples,examples,test,tests,doc,.tox
+ignore=_vendor,_generated,samples,examples,test,tests,doc,.tox, storage #Ignore sdk/storage
```

Comment:
I'm thinking this might be too broad of an ignore. Was there any other way to disable those specific checkers for storage? Also, instead of ignoring ALL of storage, could we get it down to a more specific subset possibly? Just brainstorming so we dont have to disable pylint for all of storage.